integration/terraform/ec2/README.md

```diff
@@ -160,9 +158,7 @@ The security group(s) that the integration tests use should include the followin
 | HTTPS       | 443  | 0.0.0.0/0 |
 | HTTP        | 80   | 0.0.0.0/0 |
 | SSH         | 22   | 0.0.0.0/0 |
-| RDP         | 3389 | 0.0.0.0/0 |
-| WinRM-HTTP  | 5985 | 0.0.0.0/0 |
-| WinRM-HTTPS | 5986 | 0.0.0.0/0 |
```

Comment:
Why are these being taken out of the doc? I'm pretty sure this is necessary for terraform.

Reply:
its because we are RDP as a connection type instead of WinRM and Terraform only [recommendeds us to use WinRM (only as a last resolution)](https://www.terraform.io/language/resources/provisioners/file#destination-paths)

Reply:
**Have you confirmed that you don't need WinRM?** Your security group configured here is fully open. I know when I was configuring my integration tests, I needed these in my security group.

Reply:
I can double check this behavior again so yup

Reply:
Yes double check and use this follow ingress to confirm we only use ssh instead of `WinRM`:
```
ingress {
    from_port   = "22"
    to_port     = "22"
    protocol    = "tcp"
    cidr_blocks = ["0.0.0.0/0"]
  }
```
It will result successfully. I can apply this port here when creating  SG for more security.
However, if I used the below port, it will reject me. Therefore, I believe we do not need these two ports for connection
```
ingress {
    from_port   = "5986"
    to_port     = "5986"
    protocol    = "tcp"
    cidr_blocks = ["0.0.0.0/0"]
  }
```